src/openms/source/ANALYSIS/TOPDOWN/FLASHDeconvAlgorithm.cpp

```diff
@@ -1022,8 +951,7 @@ namespace OpenMS
         }
       }
 
-
-      if (total_signal_intensity > *std::max_element(total_harmonic_intensity.begin(), total_harmonic_intensity.end()) * 2.0) //
+      if (total_signal_intensity > 1.0 * *std::max_element(total_harmonic_intensity.begin(), total_harmonic_intensity.end())) //
```

Comment:
```suggestion
      if (total_signal_intensity > *std::max_element(total_harmonic_intensity.begin(), total_harmonic_intensity.end())) //
```

Reply:
If this is not the same, please add a comment.

Reply:
They are. I tried with different values beforehand and 1.0 is the most reasonable value. Before the calculation was not correct and I used 2.0 and now after fixing the bug 1.0 resulted in the best deconvolution.